lib/coverband/adapters/base.rb

```diff
@@ -57,22 +57,14 @@ def covered_lines_for_file(file)
       protected
 
       def split_coverage(types)
-        original_type = type
         types.reduce({}) do |data, type|
```

Comment:
Avoid setting any state by allowing the method param on get_report. Great solution!